CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
 # Changelog
 
 ## [Unreleased]
+### Added
+- `mapper::Identity` is added as a built-in mapper ([#48]).
```

Comment:
Sorry, but you also need to add the URL. Please add it to the bottom of this file.